tests/foreman/cli/test_docker.py

```diff
@@ -60,7 +58,7 @@ def _make_docker_repo(product_id, name=None, upstream_name=None):
     return make_repository({
         'content-type': REPO_CONTENT_TYPE,
         'docker-upstream-name': upstream_name or REPO_UPSTREAM_NAME,
-        'name': name or gen_string(gen_choice(STRING_TYPES), 15),
+        'name': name or generate_strings_list(15, ['numeric', 'html']),
```

Comment:
Regression is introduced here as command definitely expected single string, not a list